src/tpm_codec.c

```diff
@@ -1075,62 +1075,52 @@ TSS_DispatchCmd(
         {
             result = TPM_RC_SUCCESS;
 
```

Comment:
This diff looks large, but all I did was remove 1078-1086 and a closing brace at the end.